default_app/main.ts

```diff
@@ -145,13 +146,57 @@ function startRepl () {
     process.exit(1);
   }
 
-  // prevent quitting
+  // Prevent quitting.
   app.on('window-all-closed', () => {});
 
-  const repl = require('repl');
-  repl.start('> ').on('exit', () => {
+  const colors = { GREEN: '32', MAGENTA: '35' };
```

Comment:
Not sure about using these colors as they are not great for accessibility.

Reply:
hmm good point - do you know what might perhaps be better here? i mostly just want them to stand out 🤔

Reply:
It's all about contrast.  Looks like a green against a black background is ok.  Why not use the same color for both?  You can play around with color contrast here: https://webaim.org/resources/contrastchecker/